TKGame/Level Editor Content/Stages/defaultStage.json

```diff
@@ -2,8 +2,8 @@
   {
```

Comment:
We will need some kind of map of levels in the future possibly. Looks good for now though.